corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -219,7 +219,7 @@ def process_change(self, change):
         pass
 
     @abstractmethod
-    def fire_change_processed_event(self, change, context):
```

Comment:
Got rid of most of `fire_change_processed` methods and renamed/refactored it into `update_checkpoint`, since that's the only thing that `fire_change_processed` was doing